src/re_pressed/impl.cljs

```diff
@@ -11,13 +11,10 @@
 
 (def max-record 25)
 
-(def modifier-keys #{16 ;; shift
-                     17 ;; ctrl
-                     18 ;; alt
-                     91 ;; windows (i.e. meta) ... TODO: apple key?
-                     })
-
-
+(def modifier-keys #{"Control"
+                     "Shift"
+                     "Alt"
+                     "Meta"})
```

Comment:
An example of the value returned by the `.key` event for when these modifier keys are being pressed